ui/app/hooks/useCancelTransaction.js

```diff
@@ -19,27 +25,60 @@ import { getConversionRate, getSelectedAccount } from '../selectors';
  * @return {[boolean, Function]}
  */
 export function useCancelTransaction(transactionGroup) {
-  const { primaryTransaction, initialTransaction } = transactionGroup;
+  const { primaryTransaction } = transactionGroup;
   const gasPrice = primaryTransaction.txParams?.gasPrice?.startsWith('-')
     ? '0x0'
     : primaryTransaction.txParams?.gasPrice;
-  const { id } = initialTransaction;
+  const transaction = primaryTransaction;
   const dispatch = useDispatch();
   const selectedAccount = useSelector(getSelectedAccount);
   const conversionRate = useSelector(getConversionRate);
+  const defaultNewGasPrice = addHexPrefix(
+    multiplyCurrencies(gasPrice, 1.2, {
```

Comment:
Was this change left in from testing? This used to be 1.1

```suggestion
    multiplyCurrencies(gasPrice, 1.1, {
```